### PR TITLE
extended fiesta-ci to check js spec files for the `.only()` statements

### PR DESCRIPTION
**Pull request by asia-t**

### CODE REVIEW

File: bin/fiesta-run.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 const argv = require('minimist')(process.argv.slice(2));
-const validateToolVersion = require('../src/validate_tool_versions')
+const validateToolVersion = require('../src/validate_pull_request')
 
 const apiEndpoint = argv._[0]
 const user = argv._[1]
 const repo = argv._[2]
 const commitHash = argv._[3]
 const number = argv._[4]
+const projectType =argv._[5]
```
Comment (ertrzyiks) [inline bin/fiesta-run.js]: unused?

File: src/validate_pull_request.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    function hasOnlyDirective(file) {
+        return file.patch.match(/describe\.only|it\.only/)
+    }
+
+    var originalFiles;
+
+    var hasOnlyDirectiveTest = function () {
+        return Promise.resolve()
+            .then(() => {
+                return originalFiles.filter(isJsSpec)
```
Comment (ertrzyiks) [inline src/validate_pull_request.js]: you could pass originalFiles to this function

File: bin/fiesta-run.js (hunk 1 above)
Comment (asia-t) [inline bin/fiesta-run.js]: @ertrzyiks Yup, forgot about it!

**Comment by asia-t** [comment]
Got one 👍 on the changes :D

**Pull request merged**